README.md

```diff
@@ -0,0 +1,175 @@
+# Open Context for Node.js
+
+This is a simple library for providing Distributed Context Problem
```

Comment:
May want to reference prior art like https://twitter.github.io/finagle/guide/Contexts.html